src/nodejs/controllers/proxy.js

```diff
@@ -84,8 +84,16 @@ module.exports.formFindById = function formFindById(req, res, next) {
 };
 
 module.exports.formPut = function formPut(req, res, next) {
```

Comment:
I think this is only included in the PR due to creating this branch off the EDPUB-1391 branch. In the future, it'll be best just to branch off develop then work through merge conflicts as they pop up.
Hopefully that will help isolate changes and make review a bit easier

Reply:
Yes, I agree, and I will keep this in mind next time.